tests/loggers/test_mlflow_logger.py

```diff
@@ -64,6 +64,7 @@ def test_mlflow_experiment_set_up_correctly(tmp_path):
     test_mlflow_logger.post_close()
 
 
+@pytest.mark.cpu
```

Comment:
what was the issue here/why do we only want to run this on cpu?

Reply:
When fetching the params, you get `num_gpus_per_node` instead of `num_cpus_per_node` which causes an AssertionError.

Reply:
ah ok, thats fine, thanks! Since its marked as cpu only now, can you please pass `device='cpu'` to the `Trainer` construction?

Reply:
Done.